pom.xml

```diff
@@ -85,8 +85,8 @@
   <com.github.ben-manes.caffeine.version>3.0.1</com.github.ben-manes.caffeine.version>
   <org.hibernate.hibernate.version>5.6.7.Final</org.hibernate.hibernate.version>
   <com.vladmihalcea.hibernate.types.version>2.17.1</com.vladmihalcea.hibernate.types.version>
-  <com.h2database.h2.version>1.4.197</com.h2database.h2.version>
-  <org.postgresql.postgresql.version>42.4.0</org.postgresql.postgresql.version>
+  <com.h2database.h2.version>2.1.214</com.h2database.h2.version>
+  <org.postgresql.postgresql.version>42.4.2</org.postgresql.postgresql.version>
```

Comment:
Is this version tied to the H2 Database version?

Reply:
No, the Postgres and H2 versions are orthogonal. There's some loose dependency between their versions and the Hibernate version but it should be quite stable.

Reply:
Would 42.3.3 work for now? It's what's used in Quarkus 2.7.